packages/server/modules/auth/rest/index.js

```diff
@@ -40,7 +40,7 @@ module.exports = (app) => {
       return res.redirect(`${app.redirectUrl}?access_code=${ac}`)
     } catch (err) {
       sentry({ err })
-      debug('speckle:errors')(err)
+      debug('speckle:error')(err)
```

Comment:
i'd just store this logger in '@/modules/shared/utils/logger' for DRYness, but that's not a big deal

Reply:
yeah i know, but given we want to move away from debug anyhow, i did not want to put in the effort. Made the least amount of changes, to remove the inconsistency between `error` and `errors`